### PR TITLE
Josh dev

### PR DESCRIPTION
**Pull request by JoshuaWarejko**
- Delete user stuff and minor styling edits.

### CODE REVIEW

**Comment by torch2424** [comment]
Is this ready? :)

**Comment by JoshuaWarejko** [comment]
@torch2424 Yeah we good here.

File: frontend/src/app/pages/account/account.ts [hunk 1]
```diff
@@ -54,14 +59,36 @@ export class AccountPage implements OnInit {
...
+                    this.accountUpdateSuccess = null;
+                }, 5000);
             }
         }, (reason) => {
 
         });
     }
 
+
+    //experimental code TODO: Add relevant info so it doesn't crash everything
```
Comment (torch2424) [inline frontend/src/app/pages/account/account.ts]: 😂

Comment (JoshuaWarejko) [reply frontend/src/app/pages/account/account.ts]: haha I think that was Kyle's comment! Classic 😂

**Pull request merged**